set.mm

```diff
@@ -531826,6 +531823,21 @@ to being a member of a class abstraction (that every class is equal to a
                 ( A e. B <-> ( A e. _V /\ ph ) ) ) $=
     ( wcel cvv elex biadani ) BCDBEDABCFG $.
 
+  $( A set is a binary relation if and only if it belongs to the powerclass of
+     the cartesian square of the universal class.  (Contributed by BJ,
+     16-Dec-2023.) $)
+  relelpw $p |- ( A e. V -> ( Rel A <-> A e. ~P ( _V X. _V ) ) ) $=
```

Comment:
This is very similar to ~ elrelsrel (except that it does not use the associated nullary class constant ~ df-rels as recommended in point 2 of our Guidelines).